src/connectors/hierarchical-menu/connectHierarchicalMenu.js

```diff
@@ -97,6 +98,26 @@ export default function connectHierarchicalMenu(renderFn, unmountFn) {
     const [hierarchicalFacetName] = attributes;
 
     return {
+      isShowingMore: false,
+
+      // Provide the same function to the `renderFn` so that way the user
+      // has to only bind it once when `isFirstRendering` for instance
+      toggleShowMore() {},
+      cachedToggleShowMore() {
```

Comment:
Seems like this is a new way to write this pattern? Seems fine to me, but maybe at a later point we can make that consistent across all connectors?